account_multicurrency_revaluation/model/account.py

```diff
@@ -51,28 +48,28 @@ class AccountAccount(orm.Model):
                            "balance",
     }
 
-    def _revaluation_query(self, cr, uid, ids,
-                           revaluation_date,
-                           context=None):
-
-        lines_where_clause = self.pool.get('account.move.line').\
-            _query_get(cr, uid, context=context)
+    @api.multi
+    def _revaluation_query(self, revaluation_date):
+        context = self._context
```

Comment:
prefer the use of `self.env.context`

BTW can't it be used in `_query_get()` ?